cmd/run.go

```diff
@@ -369,6 +339,14 @@ a commandline interface for interacting with it.`,
 			logger.Warn("No data generated, because no script iterations finished, consider making the test duration longer")
 		}
 
+		reportWg := &sync.WaitGroup{}
```

Comment:
Given that this is now a bit blocking, I think we should move it as early as possible. So, maybe right above `if quiet || !stdoutTTY {` ?